tools/fill_db.py

```diff
@@ -0,0 +1,276 @@
+#!/usr/bin/env python3
```

Comment:
since it's multiple files here, I suggest to create a dedicated folder "tools/eviction" and put README.md there that explains how you use these files to create the eviction experiment.

Reply:
ok